pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -75,14 +75,11 @@ def configure_checkpoint_callbacks(self, checkpoint_callback: Union[ModelCheckpo
         if not self._trainer_has_checkpoint_callbacks() and checkpoint_callback is True:
             self.trainer.callbacks.append(ModelCheckpoint(dirpath=None, filename=None, mode='min'))
 
-    def configure_progress_bar(self, refresh_rate=1, process_position=0):
-        # smaller refresh rate on colab causes crashes, warn user about this
-        if os.getenv('COLAB_GPU') and refresh_rate < 20:
-            rank_zero_warn(
-                "You have set progress_bar_refresh_rate < 20 on Google Colab. This"
-                " may crash. Consider using progress_bar_refresh_rate >= 20 in Trainer.",
-                UserWarning
-            )
+    def configure_progress_bar(self, refresh_rate=None, process_position=0):
+        if os.getenv('COLAB_GPU') and refresh_rate is None:
```

Comment:
is this env always set for colab? also on TPU?

Reply:
it's `'0'` otherwise

Reply:
@rohitgr7 not according to my quick testing.
https://colab.research.google.com/drive/1_W9Dk-cxbBZmg6R2tNrqu3gzpjcEPjdb?usp=sharing
I get True back on CPU, GPU, and TPU even after restarting the runtime.